comp/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -351,24 +351,26 @@ private void configureDriverBindings() {
                 new JoystickButton(driverXbox, XboxController.Button.kA.value)
                                 .onTrue(intakeCommands.intakeOut());
 
+                new JoystickButton(driverXbox, XboxController.Button.kB.value)
+                                .onTrue(intakeCommands.intakeJustIn());
+
                 new JoystickButton(driverXbox, XboxController.Button.kBack.value)
                                 .onTrue(swerveCommands.orbitReefCenter());
 
                 new JoystickButton(driverXbox, XboxController.Button.kX.value)
                                 .onTrue(intakeCommands.intakeOut());
 
+                new Trigger(() -> driverXbox.getPOV() == 270)
+                                .whileTrue(elevator.goToSetpoint((isPanel) ? Constants.SetpointConstants.REEF_L2
+                                                : Constants.SetpointConstants.REEF_L2));
                 new Trigger(() -> driverXbox.getPOV() == 0)
-                                .onTrue(swerveCommands.lookAtAngle(0));
-                new Trigger(() -> driverXbox.getPOV() == 315)
-                                .onTrue(swerveCommands.lookAtAngle(60));
-                new Trigger(() -> driverXbox.getPOV() == 225)
-                                .onTrue(swerveCommands.lookAtAngle(120));
+                                .whileTrue(elevator.goToSetpoint((isPanel) ? Constants.SetpointConstants.REEF_L3
```

Comment:
Can we remove the isPanel condition? Both are the same value and it looks like we hard code this variable to false.